crates/ruff_linter/src/rules/ruff/rules/assert_with_print_message.rs

```diff
@@ -66,7 +66,8 @@ pub(crate) fn assert_with_print_message(checker: &mut Checker, stmt: &ast::StmtA
             diagnostic.set_fix(Fix::unsafe_edit(Edit::range_replacement(
                 checker.generator().stmt(&Stmt::Assert(ast::StmtAssert {
                     test: stmt.test.clone(),
-                    msg: print_arguments::to_expr(&call.arguments).map(Box::new),
+                    msg: print_arguments::to_expr(&call.arguments, checker.stylist().quote())
```

Comment:
```suggestion
                    msg: print_arguments::to_expr(&call.arguments, checker.preferred_quote())
```

Or you could use `checker.default_string_flags()` and pass a `StringLiteralFlags` instance down into the lower functions rather than passing a `Quote` down -- either works!

Reply:
Ooh good catch. I went for the `StringLiteralFlags` approach, but I'm happy with either one too.